### ISSUE TITLE
Container runs only with ipv6 - unable to run ipv4

### ISSUE THREAD

**Issue by axeII** [linuxserver/docker-unifi-arm64#19]
Unable to run ipv4 this dockerimage 

[linuxserverurl]: https://linuxserver.io
[![linuxserver.io](https://raw.githubusercontent.com/linuxserver/docker-templates/master/linuxserver.io/img/linuxserver_medium.png)][linuxserverurl]
	

I am trying to run docker container. It runs but only on ipv6. I can't run it only ipv4 so I can access it on LAN.

Linux master 4.4.126-rockchip-ayufan-239 #1 SMP Sun May 27 18:38:24 UTC 2018 aarch64 GNU/Linux

```
tcp6       0      0 :::8080                 :::*                    LISTEN      -                   
tcp6       0      0 :::8880                 :::*                    LISTEN      -                   
tcp6       0      0 :::80                   :::*                    LISTEN      -                   
tcp6       0      0 :::8081                 :::*                    LISTEN      -                   
tcp6       0      0 :::51413                :::*                    LISTEN      -                   
tcp6       0      0 :::22                   :::*                    LISTEN      -                   
tcp6       0      0 :::8443                 :::*                    LISTEN      -                   
tcp6       0      0 :::443                  :::*                    LISTEN      -                   
tcp6       0      0 :::6789                 :::*                    LISTEN      -                   
tcp6       0      0 :::8843                 :::*                    LISTEN    
```

**Comment by j0nnymoe** [comment]
How did you install docker? I've seen this happen when not using the https://get.docker.com/ script.

**Comment by axeII** [comment]
Umm `apt-get install docker` ? Well I can't really remember but it's possible I did by wrong way. I'll try the script @j0nnymoe !

**Comment by axeII** [comment]
@j0nnymoe 
Ok I did install docker again using get.docker.com script. Still same problem.

This is the command I am using:
```
docker create \
  --name=unifi \
  -v /home/master/unifi:/config \
  -e PGID=1001 -e PUID=1001  \
  -p 3478:3478/udp \
  -p 10001:10001/udp \
  -p 8080:8080 \
  -p 8081:8081 \
  -p 8443:8443 \
  -p 8843:8843 \
  -p 8880:8880 \
  -p 6789:6789 \
  lsioarmhf/unifi-aarch64
```

**Comment by sparklyballs** [comment]
fresh pull of the image on my pine64 and then install net-tools via apt, and running netstat -tulpn

```root@007c1823862c:/usr/lib/unifi# netstat -tulpn
Active Internet connections (only servers)
Proto Recv-Q Send-Q Local Address           Foreign Address         State       PID/Program name
tcp        0      0 0.0.0.0:6789            0.0.0.0:*               LISTEN      -
tcp        0      0 0.0.0.0:8843            0.0.0.0:*               LISTEN      -
tcp        0      0 127.0.0.1:27117         0.0.0.0:*               LISTEN      -
tcp        0      0 0.0.0.0:8880            0.0.0.0:*               LISTEN      -
tcp        0      0 0.0.0.0:8080            0.0.0.0:*               LISTEN      -
tcp        0      0 0.0.0.0:8443            0.0.0.0:*               LISTEN      -
udp        0      0 0.0.0.0:10001           0.0.0.0:*                           -
udp        0      0 172.17.0.2:49516        0.0.0.0:*                           -
udp        0      0 0.0.0.0:3478            0.0.0.0:*                           -```

**Comment by axeII** [comment]
Hmm do you have ipv6 disabled? @sparklyballs

**Comment by sparklyballs** [comment]
doesn't look like it here, running netstat on the host

```pine64@Docker-Slave-Pine64-0:~$ netstat -tulpn
(Not all processes could be identified, non-owned process info
 will not be shown, you would have to be root to see it all.)
Active Internet connections (only servers)
Proto Recv-Q Send-Q Local Address           Foreign Address         State       PID/Program name
tcp6       0      0 :::22                   :::*                    LISTEN      -
udp        0      0 0.0.0.0:68              0.0.0.0:* ```

**Comment by axeII** [comment]
Something is wrong with sysctl settings I can't even disable ipv6.

**Comment by sparklyballs** [comment]
that's a host issue...

**Comment by axeII** [comment]
Ok but do you have any idea why would docker container force ipv6?

**Comment by sparklyballs** [comment]
docker daemon settings
host networking setup
it's thursday !!!

**Comment by axeII** [comment]
So to end this story... 
...I added this line to sysctl.conf `net.ipv4.ip_forward = 1`. Even though `netstat` still showed that dockers runs only on ipv6 + port, `nmap localhost` showed that I have ports 8080/8443 open. So basically docker did run however page didn't load. 

Docker log showed java permission error. ~I recreated docker as root and it now runs fine.~
Edit: Folder that syncs data with container must be owned by `www-data`. 

Thanks for this and cheers.
